frontend/src/views/Pages/AccountOverview/GddTransactionList.vue

```diff
@@ -24,8 +24,7 @@
                 {{ $t('form.memo') }}
               </small>
             </div>
-            <div v-if="decay">
-              <br />
+            <div v-if="decay" class="text-sm">
```

Comment:
Is this change wanted or by accident?

Reply:
as can be seen in the picture ... when the transaction list is queried ... the decay information comes first. 
this has the TYPE 'decay'.  
when this first element is created, there should be no empty line. 
maybe there is a nicer way to solve this. but i have it clean for now. 
![FireShot Capture 188 - Gradido Account - localhost](https://user-images.githubusercontent.com/1324583/128979284-d9da86b7-9120-45e8-a7a8-854d76de0704.png)